CONTRIBUTING.md

```diff
@@ -52,6 +52,9 @@ MAVEN_OPTS='--add-opens java.base/java.lang=ALL-UNNAMED --add-opens java.base/ja
 
 ### Running the Yarn frontend build
 
+> [!TIP]
+> If you already have Node.js installed, you do not need to change your path. Just run `corepack enable` and Yarn will be setup for you.
```

Comment:
Remember that "set up" is a verb, while "setup" is a noun:

https://www.ssw.com.au/rules/avoid-common-mistakes/